### PR TITLE
Capping Revert 

### PR DESCRIPTION
**Pull request by Froggg**
- !spray/!annex no longer exists
 - !progress no longer exists
 - Capture status is displayed in !look/!survey (only in survey for subzones)
 - Capping happens passively when a gangster is standing in a capturable district
 - A gangster must have at least 200k slime to capture
 - If gangsters from multiple gangs are present, capturing will be at a standstill until only one gang remains
 - A district will not decay, be de-captured, or be reinforced, when surrounded by allied districts
 - S/A/B/C capture times with one gangster and no buffs are 4/2/1/0.5 hours
 - Every gangster in a district contributes to capping. More Gangsters = Faster Capping
 - Lone Wolf will double capture speed when there are no other gangsters present
 - Patriot will multiply a gangster's contribution to capping speed by 1.5, stacks with lone wolf
 - Nerves of Steel no longer affects capturing
 - Unnatural Charisma adds capture speed equivalent to a single base gangster on top of the multipliers, so no stacking
 - Mutation descriptions and help text have been changed accordingly
 - You may now hold real weapons and defend yourself while capturing
 - Painting weapons don't really do anything but suck now

### CODE REVIEW

**Comment by aristotlepenguin** [comment]
were original calling times 4/1/2/.5? i seem to remember them taking 15 minutes max for C rank solo. If not then ignore me
also, we'll probably find something new for Nerves of Steel to do, but until then could you nerf its mutation level down to 1?
the other changed ones can be adjusted as well if you think they need it
I'll look over a few more things, seems pretty good so far

**Comment by aristotlepenguin** [comment]
also, check move.py line 531 under utils. could you get rid of the capping time messages we're sending to gang bases?

File: ew/cmd/district/districtcmds.py [hunk 1]
```diff
@@ -35,8 +35,8 @@ async def capture_progress(cmd):
     else:
         response += "Nobody has staked a claim to this district yet."
 
-    response += "\n\n**Current influence: {:,}**\nMinimum influence: {:,}\nMaximum influence: {:,}\nPercentage to maximum influence: {:,}%".format(abs(district_data.capture_points), int(ewcfg.min_influence[district_data.property_class]), int(ewcfg.limit_influence[district_data.property_class]),
-                                                                                                                                                   round((abs(district_data.capture_points) * 100 / (ewcfg.limit_influence[district_data.property_class])), 1))
+    response += "\n\n**Current influence: {:,}**\nMaximum influence: {:,}\nPercentage to maximum influence: {:,}%".format(abs(district_data.capture_points), int(ewcfg.max_capture_points[district_data.property_class]),
```
Comment (aristotlepenguin) [inline ew/cmd/district/districtcmds.py:38]: just curious, was this just a change of plans or is this function still being used? the patchnotes say it was removed

**Comment by Froggg** [comment]
> 
> 
> were original calling times 4/1/2/.5? i seem to remember them taking 15 minutes max for C rank solo. If not then ignore me also, we'll probably find something new for Nerves of Steel to do, but until then could you nerf its mutation level down to 1? the other changed ones can be adjusted as well if you think they need it I'll look over a few more things, seems pretty good so far

Got the nerf and the spray time alerts fixed. And yeah, the original times were different but they got increased over time, this is what they were towards the end of it according to what patchnotes I could find, and what I was able to verify with other players who were around for it

**Pull request merged**